.github/workflows/test.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        go: [1.20.x]
```

Comment:
Would this include patch releases or only major releases>

Reply:
This also includes patch releases

https://github.com/Enzime/age-plugin-tpm/pull/1/checks#step:2:12

Reply:
Awesome